spec/models/project_spec.rb

```diff
@@ -33,7 +33,7 @@
       expect(subject).to_not be_valid
     end
 
-    it "should not accept invalid Pivotal Tracker URL" do
+    it 'should not accept invalid Pivotal Tracker URL' do
```

Comment:
the `s` format is not invalid, Pivotal tracker will automatically redirect those links to the `n` format